cert/spec.go

```diff
@@ -126,7 +132,7 @@ func (spec *Spec) Identity() (*core.Identity, error) {
 	return ident, nil
 }
 
-func newSpecFromPath(path string) (*Spec, error) {
+func newSpecFromPath(path string, defaultServiceManager string) (*Spec, error) {
```

Comment:
Why is `defaultServiceManager` being passed in here? I don't see it used in the function.

Reply:
That's dead code from me refactoring, I'll remove it.